### PR TITLE
PLAT-122717: Fixed picker current value text position change in horizontal

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

**Checklist**
 I have read and understand the contribution guide
 A CHANGELOG entry is included

**Issue Resolved / Feature Added**
Agate - Renamed classes .seCondaryItemIncrement and .secondaryItemDecrement to a common name
- Added the same width as .itemIncrement for the .secondaryItem  for horizontal picker to fix the descibed issue

**Links**
PLAT-122717

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: internal/Picker/Picker.module.less [hunk 1]
```diff
@@ -95,7 +95,9 @@
 		&.horizontal {
 			.item,
 			.itemDecrement,
-			.itemIncrement {
+			.itemIncrement,
+			.secondaryItemIncrement,
+			.secondaryItemDecrement {
```
Comment (ybsung) [inline internal/Picker/Picker.module.less:100]: Could we rename `secondaryItemIncrement` and `secondaryItemDecrement` as `secondaryItem` and use only `secondaryItem` for both secondary increment and decrement item? I think that `secondaryItemIncrement` and `secondaryItemDecrement` have the same rule.

**Comment by ybsung** [comment]
We need to update [CHANGELOG.md](https://github.com/enactjs/agate/blob/develop/CHANGELOG.md) because of fixing our component.
```Suggestion
# Change Log

The following is a curated list of changes in the Enact sandstone module, newest changes on the top.

## [unreleased]
 
### Fixed
- `agate/Picker` to fix the picker item width in horizontal

## [1.0.0] - 2020-10-14

Initial release.
```

File: internal/Picker/Picker.module.less (hunk 1 above)
Comment (daniel-stoian-lgp) [reply internal/Picker/Picker.module.less:100]: good idea 👍

**Review by ybsung — APPROVED** [review]
LGTM

File: internal/Picker/Picker.module.less (hunk 1 above)
Comment (MikyungKim) [reply internal/Picker/Picker.module.less]: Oh, I'm sorry but I think it's not a good idea.
For future reference, I think the previous class names are clearer.
Would you please rollback the change you made here? I think the first commit looks good.

Comment (daniel-stoian-lgp) [reply internal/Picker/Picker.module.less:100]: @MikyungKim ,I reverted this change.

**Pull request merged**